orb.yml

```diff
@@ -62,14 +62,16 @@ commands:
           name: Check services are up
           command: |
             cd $E2E_WORKING_DIRECTORY/behave
-            docker-compose build
+            docker-compose build start_applications
             docker-compose run start_applications
             docker-compose exec clabackend bin/create_db.sh
       - run:
           name: Run behave tests
           command: |
             cd $E2E_WORKING_DIRECTORY/behave
             docker-compose run --name cla-end-to-end cla-end-to-end
+          environment:
```

Comment:
I don't think you need this as the environment variable is being provided by FALA's repo.